package.json

```diff
@@ -21,7 +21,7 @@
   },
   "dependencies": {
     "@dargmuesli/nuxt-cookie-control": "^7.2.0",
-    "@geode/opengeodeweb-front": "^6.1.0",
+    "@geode/opengeodeweb-front": "^6.1.1",
```

Comment:
If the carousel was new, it should have been a new feature and not a fix

Reply:
Indeed, I'll be more careful next time